R-package/R/lgb.train.R

```diff
@@ -238,7 +238,7 @@ lgb.train <- function(params = list(),
   # If user supplied early_stopping_rounds, add the early stopping callback
   if (using_early_stopping) {
     callbacks <- add.cb(
-      callbacks
+      callbacks = callbacks
```

Comment:
```suggestion
      cb_list = callbacks
```

This is not the correct argument. Please see the definition in https://github.com/microsoft/LightGBM/blob/785eb60c7864f195230d70bc817f53feb6d46a81/R-package/R/callback.R#L409. Can you please change this to `cb_list`, and add the keyword argument `cb` on the line right below it?